connect/src/wormholeTransfer.ts

```diff
@@ -41,64 +41,79 @@ export enum TransferState {
 }
 
 // Base type for common properties
-interface BaseTransferReceipt<PN extends ProtocolName, SC extends Chain, DC extends Chain> {
+export interface BaseTransferReceipt<PN extends ProtocolName, SC extends Chain, DC extends Chain> {
```

Comment:
does this need to be exported?

Reply:
yes because otherwise it complains that `track` leaks a private type

```
error TS4050: Return type of public static method from exported class has or is using name 'BaseTransferReceipt' from external module "/Users/artur/work/connect-sdk/connect/src/wormholeTransfer" but cannot be named.
```